mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -978,7 +900,7 @@ bool CMomentumPlayer::CanTeleport()
     return true;
 }
 
-void CMomentumPlayer::ManualTeleport(const Vector *newPosition, const QAngle *newAngles, const Vector *newVelocity)
+void CMomentumPlayer::ManualTeleport(const Vector *newPosition, const QAngle *newAngles, const Vector *newVelocity, bool bStopTimer)
```

Comment:
```suggestion
void CMomentumPlayer::ManualTeleport(const Vector *newPosition, const QAngle *newAngles, const Vector *newVelocity, bool bStopTimer /*= true*/)
```

Reply:
Ah I see what you meant here. I'll do this for new functions going forward as well.